gapis/api/vulkan/api/copy_clear_commands.api

```diff
@@ -618,12 +772,12 @@ cmd void vkCmdClearAttachments(
 
 sub void dovkCmdResolveImage(ref!vkCmdResolveImageArgs args) {
   for _ , _ , r in args.ResolveRegions {
-    srcRange := VkImageSubresourceRange(r.srcSubresource.aspectMask,
-                                        r.srcSubresource.mipLevel, 1,
-                                        r.srcSubresource.baseArrayLayer, r.srcSubresource.layerCount)
-    dstRange := VkImageSubresourceRange(r.dstSubresource.aspectMask,
-                                        r.dstSubresource.mipLevel, 1,
-                                        r.dstSubresource.baseArrayLayer, r.dstSubresource.layerCount)
+    srcRange := VkImageSubresourceRange(               r.srcSubresource.aspectMask,
```

Comment:
Something weird with the spacing here

Reply:
wow, looks like the format tool goes crazy.

Reply:
Fixed in 2a680ed